editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1896,7 +1942,7 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	load->set_shortcut_context(frame_list);
 	load->set_shortcut(ED_SHORTCUT("sprite_frames/load_from_file", TTR("Add frame from file"), KeyModifierMask::CMD_OR_CTRL | Key::O));
 	load_sheet->set_shortcut_context(frame_list);
-	load_sheet->set_shortcut(ED_SHORTCUT("sprite_frames/load_from_sheet", TTR("Add frames from sprite sheet"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::O));
+	load_sheet->set_shortcut(ED_SHORTCUT("sprite_frames/load_from_sheet", TTR("Add frames from sprite sheet | (Shift + Click) to load new texture"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::O));
```

Comment:
This can't be added like that, because it becomes a part of shortcut's name:
![image](https://github.com/godotengine/godot/assets/2223172/65006cf1-302a-4b25-8547-9c3ae6577700)
Instead you should use `set_tooltip_text()` for the note.
```suggestion
	load_sheet->set_shortcut(ED_SHORTCUT("sprite_frames/load_from_sheet", TTR("Add frames from sprite sheet"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::O));
	load_sheet->set_tooltip_text("Shift + Click: Load new texture.")
```
btw the way it is implemented, if a shortcut includes Shift (like the default one), the file dialog will be opened when you press it.